thirdparty/libpng/CMakeLists.txt

```diff
@@ -28,15 +28,15 @@ endif()
 ko_write_gitclone_script(
     GIT_CLONE_SCRIPT_FILENAME
     https://github.com/glennrp/libpng.git
-    v1.6.34
+    v1.6.35
     ${SOURCE_DIR}
 )
 
 include(ExternalProject)
 ExternalProject_Add(
     ${PROJECT_NAME}
     DOWNLOAD_COMMAND ${CMAKE_COMMAND} -P ${GIT_CLONE_SCRIPT_FILENAME}
-    PATCH_COMMAND sh autogen.sh
+    PATCH_COMMAND ""
```

Comment:
<3 fixing things by removing stuff

Reply:
Yeah, my first reflex was "fuck it, let's autoreconf -fi ourselves"... and then I checked the repo at that tag, and people rejoiced! :D.